### PR TITLE
fix: bad prop causing bulk mgmt errors to not show

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Fix a prop name that was causing error banners not to show for bulk management file uploads.

JIRA: [EDUCATOR-5802](https://openedx.atlassian.net/browse/EDUCATOR-5802)

**What changed?**

- Fixed a prop name

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

1. Go to Gradebook configured for bulk grades management
2. Go to the Bulk Management tab
3. Upload an intentionally malformatted CSV (e.g. [NASA's Exoplanet Archive data](https://exoplanetarchive.ipac.caltech.edu/cgi-bin/TblView/nph-tblView?app=ExoTbls&config=PSCompPars), exported as CSV is incompatible with bulk grades both because it has the wrong schema and because it exceeds our upload size limitation)

Existing behavior:
- Error banner does not show

Corrected behavior:
- Error banner shows reason for upload error: e.g. size limitation

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by jansenk — APPROVED** [review]
🚢
